core/arch/arm/kernel/sub.mk

```diff
@@ -36,6 +36,7 @@ srcs-$(CFG_ARM64_core) += misc_a64.S
 srcs-y += mutex.c
 srcs-y += wait_queue.c
 srcs-$(CFG_PM_STUBS) += pm_stubs.c
+cflags-pm_stubs.c-y += -Wno-suggest-attribute=noreturn
```

Comment:
Ha! You're right, I didn't notice the warnings on `main.c` because there are so many other warnings logged by `make C=1` :(

OK, omitting `__noreturn` is probably our best choice here.